common/lib/xmodule/xmodule/templates/problem/drag_and_drop.yaml

```diff
@@ -9,10 +9,10 @@ data: |
       In drag and drop problems, students respond to a question by dragging text or objects to a specific location on an image.
       </p>
       <p>
-      For more information, see 
-      <a href="http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/exercises_tools/drag_and_drop.html" target="_blank">
-      Drag and Drop Problem</a> in <i>Building and Running an edX Course</i>.</p>
-      <p>When you add the problem, be sure to select <strong>Settings</strong> 
+      For more information, see
+      <a href="http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/drag_and_drop_deprecated.html" target="_blank">
+      Drag and Drop Problem (Deprecated)</a> in <i>Building and Running an edX Course</i>.</p>
+      <p>When you add the problem, be sure to select <strong>Settings</strong>
         to specify a <strong>Display Name</strong> and other values that apply.</p>
       <p>You can use the following example problems as models.</p>
        <p>_____________________________________________________________________________</p>
```

Comment:
Just tested and yes, it is possible to use an `<hr />`. And yes, it is probably much better to do so.